tools/mo/openvino/tools/mo/middle/fusings.py

```diff
@@ -77,12 +77,11 @@ def find_and_replace_pattern(self, graph: Graph):
             for_graph_and_each_sub_graph_recursively(graph, fuse_linear_ops)
             for_graph_and_each_sub_graph_recursively(graph, lambda G: G.clean_up())
 
-        if not argv.disable_gfusing:
-            for_graph_and_each_sub_graph_recursively(graph, grouped_convolutions_fusing)
+        for_graph_and_each_sub_graph_recursively(graph, grouped_convolutions_fusing)
+        for_graph_and_each_sub_graph_recursively(graph, lambda G: G.clean_up())
+        if not argv.disable_fusing:
```

Comment:
`disable_fusing` needs to stay as deprecated since we have mutliple Caffe tests with it.